demos/Gtk/Gtk4Demo/PixelWindow.cs

```diff
@@ -128,10 +465,8 @@ private static void DrawFunction<TFunction>(ImageSurface imageSurface, int width
 
             for (int /* j */ x = 0; x < data_y.Length; ++x)
```

Comment:
`Parallel`?

Reply:
Here it's not good, because of the `PixelAccessor`. Could be worked around, but not for the demo which should be kept as simple as possible.